recipes/pydantic-units/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pydantic-units" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SimonBoothroyd/pydantic-units/archive/{{ version }}.tar.gz
+  sha256: a5135ca63aaa35ac245472a94951ad3d23e2004e1e40ebaebdfffa06ec0e1886
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+    - setuptools >=61.0
+    - setuptools_scm >=8
+  run:
+    - python >=3.10
+    - pydantic
+    - openmm
+
+test:
+  imports:
+    - pydantic_units
```

Comment:
Please add a pip check

Reply:
thanks @xhochy - added in `97aba24` and the CI seems to be all green!